e2e/unsubscribed.spec.ts

```diff
@@ -0,0 +1,90 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+import metaTags from '../client/i18n/locales/english/meta-tags.json';
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
import translations from '../client/i18n/locales/english/translations.json';
import metaTags from '../client/i18n/locales/english/meta-tags.json';
```

We don't need a global page variable because...